brownfield_django/settings_shared.py

```diff
@@ -65,3 +66,6 @@
 
 ADMIN_AFFIL = 'tlc.cunix.local:columbia.edu'
 DEBUG_TOOLBAR_PATCH_SETTINGS = False
+
+SERVER_EMAIL = 'brownfield@ccnmtl.columbia.edu'
```

Comment:
Just FYI: We also control the ctl.columbia.edu SES domain so sending from there should work as well.